### PR TITLE
Hard delete entities

### PR DESCRIPTION
**Pull request by greatbacon**
This PR adds an event for "hard" deleting entities.

It clarifies the previous delete functionality that creates `Tombstone` entities as a "soft" delete.

It also updates some View templates to properly render `Tombstone`s

### CODE REVIEW

File: src/lib/server/services.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const services: Map<string, Service<any, any>> = new Map(
 		createSpaceService,
 		createEntityService,
 		updateEntityService,
-		deleteEntityService,
+		softDeleteEntityService,
```
Comment (ryanatkn) [inline src/lib/server/services.ts:42]:
```suggestion
		softDeleteEntityService,
		hardDeleteEntityService,
```

File: src/lib/server/services.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ import {
 	readEntitiesService,
 	createEntityService,
 	updateEntityService,
-	deleteEntityService,
+	softDeleteEntityService,
```
Comment (ryanatkn) [inline src/lib/server/services.ts:19]:
```suggestion
	softDeleteEntityService,
	hardDeleteEntityService,
```

**Review by ryanatkn — APPROVED** [review]
Good stuff, only had a small change to register the hard delete version.

I was curious to check if the hard delete failed tests and it looks like it does. Not sure how to read the error message, may be related to cascade.

**Comment by greatbacon** [comment]
> Good stuff, only had a small change to register the hard delete version.
> 
> I was curious to check if the hard delete passes tests after registering in the services, and it looks like it fails. Not sure how to read the error message, may be related to cascade. I'm confused that there's a tie being created for the entity that's being deleted.

Probably has something to do with how parameters are being generated. I'll take a look at it.

**Pull request merged**